isolate_example/lib/page_two.dart

```diff
@@ -35,6 +35,13 @@ class InfiniteProcessPage extends StatelessWidget {
     return Column(
       mainAxisSize: MainAxisSize.min,
       children: [
+        Container(
+          child: Text(
+            'Summation Results',
+            style: TextStyle(fontSize: 25),
```

Comment:
Odds are that you can find a text style in the app's existing theme that will work here:

```
style: Theme.of(context).textTheme.pickAFontStyleFromTheListHere
```

If you can't find one that's exactly what you want, you can modify it with `copyWith`, which should be available on all the text styles in the theme.